growth/too/tasks/scheduler.py

```diff
@@ -157,7 +159,14 @@ def schedule_ztf(json_data):
                            'validity_window_mjd':
                                json_data["validity_window_mjd"],
                            'queue_type': 'list'})
-    r.raise_for_status()
+
+    if r.status_code == 200:
+        if "already exists" in r.text:
+            flash(r.text, 'danger')
+        else:
+            flash('Successfully submitted', 'success')
+    elif r.status_code == 400:
+        flash('Submission failed... contact the admins.', 'danger')
```

Comment:
Not raising an exception results in the loss of valuable diagnostic information in our logs. I am not sure if it is right to use `flash` here at all. What about just setting Flask's HTTP error status page with contact information for us?

Reply:
I am not opposed to this, but maybe easiest is @ebellm reporting the issue in the message?

Reply:
I can definitely pass more error text back in the HTTP response body, which you could then display.  Stay tuned.

Reply:
I like that idea. But I do suggest both flashing the message and also catching and logging the exception with `log.exception()`.